server/controllers/project.controller.js

```
@@ -89,14 +89,28 @@ export function getProjects(req, res) {
 
 export function getProjectsForUser(req, res) {
   if (req.params.username) {
-    User.findOne({ username: req.params.username }, (err, user) => {
-      Project.find({user: user._id}) // eslint-disable-line no-underscore-dangle
+    // Should we return all sketches when the username is p5? Maybe just public ones?
+    if (req.params.username === 'p5') {
+      Project.find({}) // eslint-disable-line no-underscore-dangle
         .sort('-createdAt')
         .select('name files id createdAt updatedAt')
         .exec((err, projects) => {
-          res.json(projects);
+          return res.json(projects);
         });
-    });
+    } else {
+      User.findOne({ username: req.params.username }, (err, user) => {
+        if (!user) {
+          return res.send(['user not found']);
```

This should return a 404 error, i.e. look something like
```javascript
return res.status(404).json({message: 'User with that username does not exist.'});
```